data/campaigns/Under_the_Burning_Suns/scenarios/04_Descending_into_Darkness.cfg

```diff
@@ -603,6 +616,21 @@
             speaker=Kaleh
             message= _ "How can he move that fast? He is faster than any goblin rider I have ever seen. It’s almost unnatural."
         [/message]
+
+        [if]
+            [variable]
+                name=sneak_up
+                boolean_equals=yes
+            [/variable]
+            [then]
+                [message]
+                    speaker=Nym
+                    message= _ "Spotted immediately! Well, I guess I spoke too soon about trying sneaking."
+                [/message]
+            [/then]
+        [/if]
+
+        {CLEAR_VARIABLE sneak_up}
```

Comment:
(Also posted on Discord) Maybe we should make a WML coding style rule for variables that depend on the events of a previous scenario, so we only clear variables like `sneak_up` in the `victory` event? That would make it easy to add more conditional dialogue, even in events that might not be triggered in all playthroughs.

Reply:
I think I understand what you're saying. This entails putting `{CLEAR_VARIABLE sneak_up} in the `victory` event, correct?

Reply:
Yes